### DISCUSSION TITLE
Problema com a propriedade p-short-logo do componente po-menu

### DISCUSSION THREAD

**Discussion by paulojunior-ufla** [po-ui/po-angular#1167] [Tire suas dúvidas] [answered]
Olá pessoal.

Adicionei a propriedade `p-short-logo` no component `po-menu`, conforme indicado na [documentação](https://po-ui.io/documentation/po-menu?view=doc). Contudo, não consigo ver mudança visual no componente, isto é, uma logomarca exibida quando o componente está colapsado, localizada na parte superior.

Segue o código utilizado:
```html
<po-menu 
    [p-menus]="menus" 
    [p-filter]="true" 
    p-logo="/assets/logo.png" 
    p-short-logo="/assets/small_logo.png">
  </po-menu>
```
Alguém poderia me ajudar a entender o que há de errado?

Desde já, muito obrigado.
Paulo.

**Comment by brunoromeiro** [comment, marked as answer]
Olá @paulojunior-ufla, tudo bem?

O `p-short-logo` aparece quando o Menu está colapsado, e para que isso aconteça é preciso que defina a propriedade `p-collapsed` como `true`.

Criei um exemplo aqui para você ver funcionando melhor.

Exemplo do código: https://stackblitz.com/edit/po-ui-xg8rex?file=src%2Fapp%2Fapp.component.html
Exemplo do menu funcionando: https://po-ui-xg8rex.stackblitz.io/

Vê se essa implementação te ajuda.
Fico a disposição.

**Reply by paulojunior-ufla** [reply]
Olá @brunoromeiro.

Espero que você esteja bem. Melhor explicação que essa, impossível!! Muito obrigado mesmo.

Abraços.
Paulo.
